src/server/users.ts

```diff
@@ -20,8 +20,13 @@ export const getUser = async (
     return null;
   }
 
+  if (!user.email) {
+    return null;
+  }
+
   return {
-    ...user,
+    uid: user.uid,
```

Comment:
Could you change uid to id so that it matches the rest of the database?

Reply:
done v/